### ISSUE TITLE
Translatable string "Sides %s and %d are tied." has hardcoded conjunct list

### ISSUE THREAD

**Issue by stevecotton** [wesnoth/wesnoth#4274]
Low-priority, as having a match end in a 3-way tie sounds a rare occurrence; the reason for logging the issue is mainly that having the heterogeneous `%s and %d` looked wrong.

https://github.com/wesnoth/wesnoth/blob/b4c65f17964617f9d5fba84117fd9cc927a25682/data/multiplayer/eras.lua#L93

If sides 1 and 2 tie, then this will have "1" in the %s and 2 in the %d.
If sides 1, 2 and 3 tie, then this will have "1, 2" in the %s and 3 in the %d.

So the translation doesn't know how many sides there are, nor can it do a non-English-style format_conjunct_list.

**Comment by stevecotton** [comment]
As well as the hardcoded conjunct list mentioned in this bug report, the same file contains this:
https://github.com/wesnoth/wesnoth/blob/b4c65f17964617f9d5fba84117fd9cc927a25682/data/multiplayer/eras.lua#L76

My feeling is that we should sort out eras.lua and a couple of others mentioned in #4436, and then we can tell translators that `%d` and `%s` are only expected in time and date format strings.

**Comment by CelticMinstrel** [comment]
There isn't an actual reason why the first example can't simply use format_conjunct_list, right?

With the second example I feel like it should be split into multiple strings, some of which end up being non-translatable. Formatting the colour of the side number doesn't need to be translated at all; you could do like `('<span foreground="#%02x%02x%02x">'):format(whatever) .. _ 'Side $side' .. '</span>'` and so on.

**Comment by stevecotton** [comment]
The colors aren't shown, and I'm wondering how long that has been broken - the Lua code has a hardcoded list of colors "1", "2", "3", ..., "9", and doesn't support "red", "green", etc. It must have worked when the color support was added to this dialog (b7a3d324a39667bb9b4613b8a98058e4884424ee, Wesnoth 1.7.1),  it presumably still worked in d2c84fd1c1f79ca302fbaa1a0ec5379d4ecaac7d (Wesnoth 1.9.10), it definitely doesn't work in 1.14.16.

With everything written in white it looks nice, readable, and if we removed the color formatting markup from the strings then the translatable strings would be simpler too. So if it's been white for ages and no-one has complained then I'd go for the simple option and remove the color formatting completely.
